ui/ember-cli-build.js

```diff
@@ -41,9 +41,11 @@ module.exports = function (defaults) {
       webpack: {
         // uncomment this to see a tree view of what ember-cli-auto-import builds
         // plugins: [new (require('webpack-bundle-analyzer').BundleAnalyzerPlugin)()],
-        // required for ansi-colors to work in the browser
-        node: {
-          process: true,
+        resolve: {
+          fallback: {
+            // required for ansi-colors to work in the browser
+            process: require.resolve('process/browser'),
+          },
```

Comment:
This doesn’t seem to work. Not sure why yet. Will take another swing at it tomorrow.

Reply:
Just realized it’s because ansi-colors expect the `process` global to be around. Going to investigate how to convince Webpack 5 to sneak it in.

Reply:
Tried using [DefinePlugin](https://webpack.js.org/plugins/define-plugin/) but it doesn’t seem to operator on dependencies.

Reply:
Figured this out in the end